share/doc/homebrew/Interesting-Taps-&-Branches.md

```diff
@@ -6,9 +6,7 @@ Homebrew has the capability to add (and remove) multiple taps to your local inst
 
 *   [homebrew/apache](https://github.com/Homebrew/homebrew-apache): A tap for Apache modules, extending OS X's built-in Apache. These brews may require unconventional additional setup, as explained in the caveats.
 
-*   [homebrew/binary](https://github.com/Homebrew/homebrew-binary): Precompiled binary formulae.
-
-*   [homebrew/boneyard](https://github.com/Homebrew/homebrew-boneyard): Formula are not deleted, they are moved here.
+*   [homebrew/boneyard](https://github.com/Homebrew/homebrew-boneyard): Formulae are not deleted, they are moved here.
```

Comment:
Maybe “Formulae from other official taps, primarily homebrew/core, are not deleted; […]”?